backend/src/data_entry/mod.rs

```diff
@@ -65,20 +65,25 @@ pub async fn polling_station_data_entry_save(
     Path((id, entry_number)): Path<(u32, u8)>,
     data_entry_request: SaveDataEntryRequest,
 ) -> Result<SaveDataEntryResponse, APIError> {
-    // only the first data entry is supported for now
-    if entry_number != 1 {
-        return Err(APIError::NotFound(
-            "Only the first data entry is supported".to_string(),
-            ErrorReference::EntryNumberNotSupported,
-        ));
-    }
-
-    if polling_station_data_entries.exists_finalised(id).await? {
-        return Err(APIError::Conflict(
-            "Cannot save data entry for a polling station that has already been finalised"
+    // Check if it is valid to save the data entry
```

Comment:
This code accepts `entry_number`s greater than 2 now, which should not be the case.